tests/E2E/test/campaigns/common_scenarios/category.js

```diff
@@ -48,9 +48,10 @@ module.exports = {
       test('should click on "Save" button', () => {
         return promise
           .then(() => client.scrollWaitForExistAndClick(CategorySubMenu.save_button, 50))
-          .then(() => client.getTextInVar(CategorySubMenu.category_number_span, "number_category"));
+          .then(() => client.getTextInVar(CategorySubMenu.category_number_span, "number_category"))
+          .then(() => tab['number_category'] = tab['number_category'].split('(')[1].split(')')[0]);
```

Comment:
Use regex instead?

Reply:
Good suggestion.
Now, we are quick fixing tests, we will do it in the next step : refactoring

Reply:
Replace it with regex